services/web-app/pages/designing/get-started.mdx

```diff
@@ -145,8 +145,6 @@ Everyone using Carbon should use core design kits as a starting point for buildi
 This brings consistency and speed, allowing teams to focus on solving more specific user tasks. Core
 design kits include elements, core components, wireframes, and guidelines.
 
-### Core design kits
-
 <FilterableDesignKitTable
   designKitsData={props.designKitsData}
```

Comment:
All items in the table should either be links (to figma design kit previews) or downloads of design kits. This should all be defined in the schema.

Reply:
@aubrey-oneal we move this to new issue